src/pages/waiting-room-to-car/cat-c/waiting-room-to-car.cat-c.page.html

```diff
@@ -31,8 +31,14 @@
             </accompaniment-card>
           </ion-col>
         </ion-row>
-        <warning-banner class="banner-space" warningText="Is the cab locked down?"></warning-banner>
-        <warning-banner warningText="Is the load secure?"></warning-banner>
+
+        <div class="banner-space" *ngIf=" pageState.displayCabLockDown$ | async ">
+          <warning-banner warningText="Is the cab locked down?"></warning-banner>
```

Comment:
Could just have the ngIf on the warning-banner instead

Reply:
done